charts/kafka/README.md

```diff
@@ -1,25 +1,11 @@
 <!---
-Copyright 2020 Crown Copyright
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-  http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-
 Originally copied from: https://github.com/helm/charts/blob/6c85be7b88748171afd17affe8b1b57c66bf66a2/incubator/kafka/README.md
 --->
 
 # Apache Kafka Helm Chart
```

Comment:
I think we should update this readme, sticking to the original format, such that it is an accurate reflection of how to use these charts.